src/python_pachyderm/client.py

```diff
@@ -482,6 +483,19 @@ def delete_all(self) -> None:
         """Delete all repos, commits, files, pipelines, and jobs. This resets
         the cluster to its initial state.
         """
+        # Try removing all identities if auth is activated.
+        try:
+            self.delete_all_identity()
+        except AuthServiceNotActivated:
+            pass
+
+        # Try deactivating auth if activated.
+        try:
+            self.deactivate_auth()
+        except AuthServiceNotActivated:
+            pass
+
+        self.delete_all_license()
```

Comment:
Does `delete_all_license()` also need to be put in a `try-except`? https://github.com/pachyderm/pachyderm/blob/28e295e6065c2adb093467c676232d87a93b497b/src/client/client.go#L705

Reply:
Ah I never hit an error when testing which is why I didn't add this, but if core does it than I'll add it to the license mixins